ui/src/app/views/workspace-share/component.ts

```diff
@@ -122,7 +126,10 @@ export class WorkspaceShareComponent implements OnInit {
             }
             this.usersLoading = false;
           }
-      );
+        );
+      } else {
+        this.usersLoading = false;
```

Comment:
Alternatively, it would probably fix this logic and be more readable to put `this.usersLoading = true` closer to the callback (inside that `if`)

Reply:
Nice catch - done.